src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator stopped working");

```

Comment:
The message in the `stopWork` method should be more informative by indicating the type of `Machine` that has stopped working, such as 'Excavator'.